.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -12,7 +12,7 @@ We would like to use GitHub for bug reports and feature requests only.
 Try our following resources:
 
 Discord - https://discord.io/neo
-offical documentation - https://docs.neo.org/docs/en-us/index.html
+official documentation - https://docs.neo.org/docs/en-us/index.html
```

Comment:
This site is no longer maintained, new documentation site is https://developers.neo.org/